roman_numeral_converter_spec.rb

```diff
@@ -0,0 +1,70 @@
+require_relative "./roman_numeral_converter.rb"
+
+describe "Roman Numeral Converter" do
+  context "roman numeral to arabic conversion" do
+    before { @converter = RomanArabicConverter.new }
```

Comment:
If we're using `describe RomanArabicConverter do` at the top of this rspec file, we get access to a variable called `described_class`, which allows us to rewrite the above into -

```ruby
before { @converter = described_class.new }
```

But it turns out that you get access to a variable called `subject` when you follow the convention `describe RomanArabicConverter do` as well. So we could delete this line entirely, and update our tests to read as -

```ruby
specify { expect(subject.convert_to_arabic("I")).to eq(1) }
```

This is described [here](https://www.relishapp.com/rspec/rspec-core/v/2-11/docs/subject/implicitly-defined-subject).